patches.json

```diff
@@ -225,5 +228,108 @@
         }
       }
     }
+  },
+  "MDVA-28191": {
+    "magento/magento2-base": {
+      "Fixes the issue where no payment methods are loaded during order creation via Magento Admin panel.": {
+        ">=2.3.3 <2.4.2": {
+          "file": "os/MDVA-28191__fixes_no_payments_during_order_creation_in_admin_panel__2.3.5-p1.patch"
+        }
+      }
+    }
+  },
+  "MDVA-29959": {
+    "magento/magento2-b2b-base": {
+      "Fixes the issue where restricted admin user with 'Companies' permissions is not allowed to delete company account.": {
+        ">=1.1.0 <=1.1.3-p1": {
+          "file": "commerce/MDVA-29959__fixes_restricted_admin_permission_is_not_allowed_to_delete_company__2.3.3.patch"
+        }
+      }
+    }
+  },
+  "MDVA-30265": {
+    "magento/magento2-base": {
+      "Fixes the issue where shipment tracking link stops working after Invoice creation.": {
+        ">=2.3.3 <2.4.2": {
+          "file": "os/MDVA-30265__fixes_tracking_link_redirects_to_404__2.3.3.patch"
+        }
+      }
+    }
+  },
+  "MDVA-28409": {
+    "magento/magento2-base": {
+      "Fixes the issue where the 'sales_clean_quotes' cron job fails with out-of-memory error when the number of expired quotes in the database is huge.": {
+        ">=2.3.4 <2.3.6 || 2.4.0": {
+          "file": "os/MDVA-28409__cleaning_old_quotes_from_sales_clean_quotes_table__2.3.4.patch"
+        },
+        ">=2.3.0 <2.3.4": {
+          "file": "os/MDVA-30593__cleaning_old_quotes_from_sales_clean_quotes_table__2.3.3.patch"
+        }
+      }
```

Comment:
I do not think MDVA-28409 and MDVA-30593 should be merged together. They are different issues 
Please compare test case between MDVA-28409 (backport of MC-34782) and MDVA-30593 (backport of MC-19798)

Reply:
Fixed